app/views/incomeSources/cease/IncomeSourceCeasedObligations.scala.html

```diff
@@ -99,57 +99,35 @@ <h2 class="govuk-label-wrapper">
             </label>
         </h2>
 
-        @if(sources.quarterlyObligationsDatesYearOne.nonEmpty) {
+
+        @if(sources.quarterlyObligationsDates.nonEmpty) {
             <div class="box-simple" id="quarterly">
                 <div class="page_headers govuk-heading-m"> @getMessage("quarterly-heading") </div>
                 <p class="govuk-body"> @getMessage("quarterly-p1") </p>
 
-                @govukTable(Table(
-                    rows = sources.quarterlyObligationsDatesYearOne.map { x =>
-                        Seq(
-                            TableRow(
-                                content = Text(s"${x.inboundCorrespondenceFrom.toLongDate} ${getMessage("to")} ${x.inboundCorrespondenceTo.toLongDate}")
-                            ),
-                            TableRow(
-                                content = Text(x.inboundCorrespondenceDue.toLongDate)
-                            )
-                        )
-                    },
-                    head = Some(Seq(
-                        HeadCell(
-                            content = Text(getQuarterlyTableHeading(sources.quarterlyObligationsDatesYearOne))
-                        ),
-                        HeadCell(
-                            content = Text(getMessage("table.heading-2"))
-                        )
-                    )),
-                    caption = None,
-                    firstCellIsHeader = false
-                ))
-
-                @if(sources.quarterlyObligationsDatesYearTwo.nonEmpty) {
-                    @govukTable(Table(
-                        rows = sources.quarterlyObligationsDatesYearTwo.map { x =>
-                            Seq(
-                                TableRow(
-                                    content = Text(s"${x.inboundCorrespondenceFrom.toLongDate} ${getMessage("to")} ${x.inboundCorrespondenceTo.toLongDate}")
+                @for(quarterlyPeriods <- sources.quarterlyObligationsDates) {
```

Comment:
Could fix: a lot of repeated code in these views, each table could be extracted into a partial and called where necessary

See views/partials for examples